roles/tendrl-gluster-integration/README.rst

```diff
@@ -2,27 +2,12 @@
  Tendrl-gluster-integration
 ============================
 
-This role installs `tendrl-gluster-integration`_ component.
+This role installs `tendrl-gluster-integration`_ component from packages.
 
-When variable ``install_from`` is set to ``packages``, the component is
-installed from rpm packages via ``yum``. On the other hand when the value is
-set to ``source``, tendrl-gluster-integration is installed from sources from
-github via ``pip``.
+This role should not be used in production playbooks because installation
+of this component is handled by `node-agent`_ as mentioned `here`_.
```

Comment:
Not really. It is kept there for emergency testing when provisioning is broken but we can remove it along with tendrl-ceph-integration.